modules/ROOT/pages/cloudhub-deployment-settings.adoc

```diff
@@ -0,0 +1,34 @@
+= Configuring CloudHub Deployment Settings
+
+To configure settings for deploying Anypoint Partner Manager to CloudHub, access the business unit and environment for which you want to deploy Partner Manager and then follow these steps:
+
+. In the *Configure a Partner Manager environment* window, enter the following general settings:
++
+[%header%autowidth.spread]
+|===
+| Setting | Description | Default 
+| Deployment Target Type | Target type for your Partner Manager deployment. Select *CloudHub 1.0*. | CloudHub 1.0
+| Runtime Version | Mule runtime engine version to run Partner Manager applications.| 4.4.0
+| CloudHub Region |Region in CloudHub is deployed. | Retrieved from the business group's Runtime Manager setting
```

Comment:
Please change "Region in CloudHub is deployed." to "Region in CloudHub you want Partner Manager runtime applications to be deployed to". Note that the region cannot be changed when there are actively deployed message flows in the environment to the previously selected CloudHub region.